viewing_party/party.py

```diff
@@ -31,19 +31,57 @@ def watch_movie(user_data, title):
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
 
+# my own work with the help of tutor
+def get_watched_avg_rating(user_data):
+    count = 0
+    average_rating = 0
+
+    for i in range(len(user_data["watched"])):
+        count += (user_data["watched"][i]["rating"])
+        average_rating = count/len(user_data["watched"])
+    return average_rating

```

Comment:
This works great! You could also consider using a a for-each loop here:
`for movie in user_data["watched"]`

Then, we could just get `movie["rating"]` instead of `user_data["watched"][i]["rating"]`, a lot easier to work with!